invenio_celery/ext.py

```diff
@@ -34,9 +35,15 @@
 class InvenioCelery(object):
     """Invenio theme extension."""
 
-    def __init__(self, app=None, **kwargs):
+    def __init__(self, app=None, entry_point_group="invenio_celery.tasks",
```

Comment:
please follow the naming `entrypoint_name ` see https://github.com/inveniosoftware/invenio-db/blob/master/invenio_db/core.py#L54